packages/safepal-wallet/src/lib/safepal-wallet.ts

```diff
@@ -0,0 +1,289 @@
+import { SignedTransaction } from "near-api-js/lib/transaction.js";
```

Comment:
I think we can import it this way, which would let us remove the near-api-js dependency.
```suggestion
import { SignedTransaction } from "@near-wallet-selector/core";
```